Extension/package.json

```diff
@@ -42,6 +42,168 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "cppWelcome",
+        "title": "Get Started with C++ Development",
+        "description": "Your first steps to set up powerful C++ tools in a lightweight, performant editor!",
+        "steps": [
+          {
+            "id": "verify.compiler.mac",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. [Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isMac",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing Clang on macOS",
+              "path": "walkthrough/install-clang-macos.md"
+            }
+          },
+          {
+            "id": "verify.compiler.linux",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isLinux",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing GCC on Linux",
+              "path": "walkthrough/install-gcc-linux.md"
+            }
+          },
+          {
+            "id": "verify.compiler.windows",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isWindows",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing MSVC on Windows",
+              "path": "walkthrough/install-compiler-windows.md"
+            }
+          },
+          {
+            "id": "open.project",
+            "title": "Open your project",
+            "description": "Open an existing folder that contains your C++ project, or create a new folder. \n[Open Folder](command:workbench.action.files.openFolder)",
```

Comment:
Is referring to "C++ project" going to confuse anyone who may associate that with VS C++ project files or users who may wonder "how do I get a C++ project" in the first place? Should it be "C++ files" and should the title be "Open your C++ folder"? Since VS Code lacks a concept of "projects" it seems clearer to remove all references to the word?

Reply:
Maybe if we show users how to create a workspace we could change all references of project to workspace.

Reply:
True. Let's follow what VS Code has in their walkthrough which does a better job addressing this:
Title: "Open your project folder"
Button: "Pick a folder"